error.go

```diff
@@ -34,18 +34,23 @@ const (
 	ErrorCodeAPIKeyExpired                          ErrorCode = "api_key_expired"
 	ErrorCodeAuthenticationRequired                 ErrorCode = "authentication_required"
 	ErrorCodeBalanceInsufficient                    ErrorCode = "balance_insufficient"
+	ErrorCodeBankAccountDeclined                    ErrorCode = "bank_account_declined"
 	ErrorCodeBankAccountExists                      ErrorCode = "bank_account_exists"
 	ErrorCodeBankAccountUnusable                    ErrorCode = "bank_account_unusable"
 	ErrorCodeBankAccountUnverified                  ErrorCode = "bank_account_unverified"
+	ErrorCodeBankAccountVerificationFailed          ErrorCode = "bank_account_verification_failed"
 	ErrorCodeBitcoinUpgradeRequired                 ErrorCode = "bitcoin_upgrade_required"
+	ErrorCodeCardDeclinedRateLimitExceeded          ErrorCode = "card_decline_rate_limit_exceeded"
 	ErrorCodeCardDeclined                           ErrorCode = "card_declined"
 	ErrorCodeChargeAlreadyCaptured                  ErrorCode = "charge_already_captured"
 	ErrorCodeChargeAlreadyRefunded                  ErrorCode = "charge_already_refunded"
 	ErrorCodeChargeDisputed                         ErrorCode = "charge_disputed"
 	ErrorCodeChargeExceedsSourceLimit               ErrorCode = "charge_exceeds_source_limit"
 	ErrorCodeChargeExpiredForCapture                ErrorCode = "charge_expired_for_capture"
+	ErrorCodeChargeInvalidParameter                 ErrorCode = "charge_invalid_params"
```

Comment:
i think this one is `charge_invalid_parameter`

Reply:
I wish I could rewind my life and understand what happened here since I copy-pasted each value by hand. Oh well.